autoscale-api/src/main/java/com/hpe/caf/api/autoscale/InstanceInfo.java

```diff
@@ -104,4 +140,20 @@ public int getShutdownPriority()
     {
         return shutdownPriority;
     }
+
+    /**
+     * @return the max time in seconds that the app has previously taken to launch
+     */
+    public int getMaxLaunchDelaySeconds()
+    {
+        return maxLaunchDelaySeconds;
+    }
+    
+    /**
+     * @return the number of instances of this application running
+     */
+    public int getInstances()
```

Comment:
Kept it called instances to more closely relate it to what marathons variable was called.

Reply:
Move JavaDoc is required here to explain the difference between this and `getTotalRunningAndStageInstances()`.  It will not be obvious to a fresh developer.